build.sbt

```diff
@@ -326,13 +326,13 @@ lazy val sshd = project
       name := "ammonite-sshd",
       libraryDependencies ++= Seq(
         // sshd-core 1.3.0 requires java8
-        "org.apache.sshd" % "sshd-core" % "1.2.0",
-        "org.bouncycastle" % "bcprov-jdk15on" % "1.56",
+        "org.apache.sshd" % "sshd-core" % "1.6.0",
```

Comment:
I'd say we should keep it, unless there's some really-truly pressing reason we need to upgrade `sshd-core`. I don't have numbers but wouldn't be surprised if there were double-digit % of Ammonite users still on Java6